<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rompar</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/res.qrc">
    <normaloff>:/mainicon_64x64.png</normaloff>:/mainicon_64x64.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>5</number>
    </property>
    <item row="1" column="0">
     <widget class="RomparGridView" name="graphicsView">
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="buttonToggleMode">
      <property name="statusTip">
       <string>Select between grid and data edit mode.</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources/res.qrc">
        <normaloff>:/toggleModeIcons/toggleModeIcons/modeEditData.png</normaloff>
        <normalon>:/toggleModeIcons/toggleModeIcons/modeEditGrid.png</normalon>:/toggleModeIcons/toggleModeIcons/modeEditData.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionBackupSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuCVOptions">
    <property name="title">
     <string>C&amp;V options</string>
    </property>
    <addaction name="actionRadiusIncrease"/>
    <addaction name="actionRadiusDecrease"/>
    <addaction name="actionDilateIncrease"/>
    <addaction name="actionDilateDecrease"/>
    <addaction name="actionErodeIncrease"/>
    <addaction name="actionErodeDecrease"/>
    <addaction name="actionBitThresholdDivisorIncrease"/>
    <addaction name="actionBitThresholdDivisorDecrease"/>
    <addaction name="actionPixelThresholdMinimumIncrease"/>
    <addaction name="actionPixelThresholdMinimumDecrease"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>&amp;Data</string>
    </property>
    <addaction name="actionDataInverted"/>
    <addaction name="actionDataLSBitMode"/>
    <addaction name="actionFindHex"/>
    <addaction name="actionSaveDataAsText"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Displa&amp;y</string>
    </property>
    <widget class="QMenu" name="menuBase_Image">
     <property name="title">
      <string>Base &amp;Image</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionImgBGBlank"/>
     <addaction name="actionImgBGOriginal"/>
     <addaction name="actionImgBGTarget"/>
    </widget>
    <addaction name="menuBase_Image"/>
    <addaction name="actionShowGrid"/>
    <addaction name="actionShowDataBinary"/>
    <addaction name="actionShowPeephole"/>
    <addaction name="actionShowData"/>
    <addaction name="actionFontIncrease"/>
    <addaction name="actionFontDecrease"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionManual"/>
    <addaction name="actionAuthors"/>
    <addaction name="actionLicense"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menu_mode">
     <property name="title">
      <string>&amp;mode</string>
     </property>
     <addaction name="actionGridEditMode"/>
     <addaction name="actionDataEditMode"/>
     <addaction name="separator"/>
     <addaction name="actionToggleMode"/>
    </widget>
    <addaction name="menu_mode"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuCVOptions"/>
   <addaction name="menuData"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRadiusIncrease">
   <property name="text">
    <string>Radius +</string>
   </property>
   <property name="toolTip">
    <string>Radius Increase</string>
   </property>
   <property name="shortcut">
    <string>Shift+A</string>
   </property>
  </action>
  <action name="actionRadiusDecrease">
   <property name="text">
    <string>Radius -</string>
   </property>
   <property name="toolTip">
    <string>Radius Decrease</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionDilateIncrease">
   <property name="text">
    <string>Dilate +</string>
   </property>
   <property name="toolTip">
    <string>Dilate Increase</string>
   </property>
   <property name="shortcut">
    <string>Shift+D</string>
   </property>
  </action>
  <action name="actionDilateDecrease">
   <property name="text">
    <string>Dilate -</string>
   </property>
   <property name="toolTip">
    <string>Dilate Decrease</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actionErodeIncrease">
   <property name="text">
    <string>Erode +</string>
   </property>
   <property name="toolTip">
    <string>Erode Increase</string>
   </property>
   <property name="shortcut">
    <string>Shift+E</string>
   </property>
  </action>
  <action name="actionErodeDecrease">
   <property name="text">
    <string>Erode -</string>
   </property>
   <property name="toolTip">
    <string>Erode Decrease</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionFontIncrease">
   <property name="text">
    <string>Font +</string>
   </property>
   <property name="toolTip">
    <string>Font Increase</string>
   </property>
   <property name="shortcut">
    <string>Shift+F</string>
   </property>
  </action>
  <action name="actionFontDecrease">
   <property name="text">
    <string>Font -</string>
   </property>
   <property name="toolTip">
    <string>Font Decrease</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionBitThresholdDivisorIncrease">
   <property name="text">
    <string>Bit Threshold Divisor +</string>
   </property>
   <property name="toolTip">
    <string>Bit Threshold Divisor Increase</string>
   </property>
   <property name="shortcut">
    <string>Shift+M</string>
   </property>
  </action>
  <action name="actionBitThresholdDivisorDecrease">
   <property name="text">
    <string>Bit Threshold Divisor -</string>
   </property>
   <property name="toolTip">
    <string>Bit Threshold Divisor Decrease</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="actionPixelThresholdMinimumIncrease">
   <property name="text">
    <string>Pixel Threshold Minimum +</string>
   </property>
   <property name="toolTip">
    <string>Pixel Threshold Minimum Increase</string>
   </property>
   <property name="shortcut">
    <string>=</string>
   </property>
  </action>
  <action name="actionPixelThresholdMinimumDecrease">
   <property name="text">
    <string>Pixel Threshold Minimum -</string>
   </property>
   <property name="toolTip">
    <string>Pixel Threshold Minimum Decrease</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionImgBGTarget">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Target</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="actionImgBGBlank">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Blank</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionImgBGOriginal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
  </action>
  <action name="actionShowGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Grid</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionShowDataBinary">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Binary</string>
   </property>
   <property name="shortcut">
    <string>Shift+H</string>
   </property>
  </action>
  <action name="actionDataInverted">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Inverted</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="actionDataLSBitMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Least Significant Bit Mode</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionShowPeephole">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Peephole</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionShowData">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Data</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionBackupSave">
   <property name="text">
    <string>&amp;Backup and Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
  <action name="actionFindHex">
   <property name="text">
    <string>&amp;Find Hex</string>
   </property>
   <property name="shortcut">
    <string>/</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>&amp;Manual</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>&amp;License</string>
   </property>
  </action>
  <action name="actionAuthors">
   <property name="text">
    <string>Aut&amp;hors</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionSaveDataAsText">
   <property name="text">
    <string>&amp;Export Data as Text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
  <action name="actionGridEditMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/res.qrc">
     <normaloff>:/toggleModeIcons/toggleModeIcons/modeEditGrid.png</normaloff>:/toggleModeIcons/toggleModeIcons/modeEditGrid.png</iconset>
   </property>
   <property name="text">
    <string>Grid Edit Mode</string>
   </property>
  </action>
  <action name="actionDataEditMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/res.qrc">
     <normaloff>:/toggleModeIcons/toggleModeIcons/modeEditData.png</normaloff>:/toggleModeIcons/toggleModeIcons/modeEditData.png</iconset>
   </property>
   <property name="text">
    <string>Data Edit Mode</string>
   </property>
  </action>
  <action name="actionToggleMode">
   <property name="text">
    <string>&amp;Toggle Mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RomparGridView</class>
   <extends>QGraphicsView</extends>
   <header>rompargridview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
